### PR TITLE
Fix various notices

### PR DESCRIPTION
**Pull request by MrPetovan**
Part of #5470

This code hasn't been tested because of the item table structure update currently running on my node for 80 minutes.

### CODE REVIEW

File: mod/photos.php [hunk 1]
```diff
@@ -1274,8 +1276,12 @@ function photos_content(App $a)
 					}
 				}
 				$edit_suffix = ((($cmd === 'edit') && $can_post) ? '/edit' : '');
-				$prevlink = 'photos/' . $a->data['user']['nickname'] . '/image/' . $prvnxt[$prv]['resource-id'] . $edit_suffix . ($order_field === 'posted' ? '?f=&order=posted' : '');
-				$nextlink = 'photos/' . $a->data['user']['nickname'] . '/image/' . $prvnxt[$nxt]['resource-id'] . $edit_suffix . ($order_field === 'posted' ? '?f=&order=posted' : '');
+				if (!is_null($prv)) {
+					$prevlink = 'photos/' . $a->data['user']['nickname'] . '/image/' . $prvnxt[$prv]['resource-id'] . $edit_suffix . ($order_field === 'posted' ? '?f=&order=posted' : '');
```
[the remark this answers is not in the document]
Comment (annando) [reply mod/photos.php]: Örgs ...

File: src/Model/Profile.php [hunk 2]
```diff
@@ -359,10 +361,9 @@ private static function sidebar($profile, $block = 0, $show_connect = true)
 				$remote_url = $r[0]['url'];
 				$message_path = preg_replace('=(.*)/profile/(.*)=ism', '$1/message/new/', $remote_url);
 				$wallmessage_link = $message_path . base64_encode($profile['addr']);
+			} else if (!empty($profile['nickname'])) {
```
[the remark this answers is not in the document]
Comment (annando) [reply src/Model/Profile.php]: This one still stands :-)

[the remark this answers is not in the document]
Comment (MrPetovan) [reply src/Model/Profile.php]: Ha ha, the impatience gt the better of you!

[the remark this answers is not in the document]
Comment (annando) [reply src/Model/Profile.php]: It had only been some "standards" problem - where your normally are fighting for - and I'm more relaxed about. So not me but you could have a problem with this :grin:

[the remark this answers is not in the document]
Comment (MrPetovan) [reply src/Model/Profile.php]: Of course, but I was traveling, so there was nothing I could do. Oh well, I suppose @Quix0r will eventually pick it up!

**Pull request merged**